src/Microsoft.Diagnostics.Monitoring.RestServer/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Diagnostics.Monitoring.RestServer
+{
+    public class Program
+    {
+        public static IWebHostBuilder CreateWebHostBuilder(IDiagnosticServices diagServices) =>
```

Comment:
If the intention is to also ship a library that can be used in an existing application then we should move this logic into the executable. ASP.NET Core libraries that plug into existing applications have 2 entry points:
- IServiceCollection extension methods
- IApplicationBuilder extension methods (middleware)
- IEndpointRouteBuilder extension methods in 3.0 (endpoint routing in 3.0)